app/views/GermlineView/components/Board/columnDefs.js

```diff
@@ -0,0 +1,69 @@
+import { formatDate } from '../../../../utils/date';
```

Comment:
could probably update the path here to the new @

Reply:
oh wait this is waiting on the other PR to be merged in ... guess we can just fix it as we go

Reply:
The other PR is just so it doesn't show as a linting error - it still works. I can add it here